README.md

```diff
@@ -35,9 +35,109 @@ Follow these steps to delete a dataspace instance and free up the corresponding
 - Provide the resources prefix that you used when you deployed your DataSpace.
 - Click on `Run workflow` to trigger to destroy your MinimumViableDataspace DataSpace.
 
-## Local development setup
-
-Please follow the instructions in [this document](system-tests/README.md) to setup a local MVD environment for development purposes.
+## Local Development Setup
+
+In addition to the services described in [system-tests/README.md](system-tests/README.md) the local development setup
```

Comment:
I think this paragraph will be the first time the reader encounters the local development setup topic. I would opt for not assuming he or she is familiar with the information available in the system test README, and adapt the wording accordingly.

Reply:
I've changed the wording. Thanks for sharing your point of view.